pytse_client/ticker.py

```diff
@@ -94,19 +99,54 @@ def last_price(self):
     def adj_close(self):
         return self.get_ticker_real_time_info_response().adj_close
 
+    @property
+    def best_demand_vol(self):
+        return self.get_ticker_real_time_info_response(
+            best_info=True).best_demand_vol
+
+    @property
+    def best_demand_price(self):
+        return self.get_ticker_real_time_info_response(
+            best_info=True).best_demand_price
+
+    @property
+    def best_supply_vol(self):
+        return self.get_ticker_real_time_info_response(
+            best_info=True).best_supply_vol
+
+    @property
+    def best_supply_price(self):
+        return self.get_ticker_real_time_info_response(
+            best_info=True).best_supply_price
+
     @property
     @functools.lru_cache()
     def ticker_page_response(self):
         return utils.requests_retry_session().get(self._url, timeout=10)
 
-    def get_ticker_real_time_info_response(self) -> RealtimeTickerInfo:
+    def get_ticker_real_time_info_response(
+        self, best_info=False) -> RealtimeTickerInfo:
```

Comment:
Why do we need best_info as an argument? we are already requesting info there is no difference between returning all the values or 2 of them.

Reply:
For retrieving 4 last arguments of real-time data, I used `.split('@')[1]` for parsing the response. Some times this part of code I mentioned raises an exception due to changes in response format. I added the best_info so that when we only need adj_close or last_info, we won't come up with this unnecessary exception… I know  it's a dirty solution (adding best_info); but in this way we end up with less raised exceptions for user.
Absolutely there are better ways of implementing this, but other ways needed a lot more change in the structure of the Ticker class and I wasn't sure about doing them.